tests/tests/gas_price.rs

```diff
@@ -270,12 +382,12 @@ async fn latest_gas_price__if_node_restarts_gets_latest_value() {
         "--starting-gas-price",
         "1000",
         "--gas-price-change-percent",
-        "10",
+        "100",
```

Comment:
Why is this different? We can probably revert?

Reply:
action: @MitchTurner